syntax = "proto3";

package test.integration.start_tls;

message StartTlsFilterConfig{
}
